packages/preview-middleware-client/src/cpe/changes/service.ts

```diff
@@ -333,7 +333,13 @@ export class ChangeService {
             if (this.eventStack.length - 1 === eventIndex) {
                 this.pendingChanges = pendingChanges.filter((change): boolean => !!change);
                 const changesRequiringReload = this.pendingChanges.reduce(
-                    (sum, change) => (change.changeType === 'appdescr_fe_changePageConfiguration' ? sum + 1 : sum),
+                    (sum, change) =>
+                        [
+                            'appdescr_fe_changePageConfiguration',
+                            'appdescr_ui_generic_app_changePageConfiguration'
```

Comment:
We also need to add this new change type to the https://github.com/SAP/open-ux-tools/blob/8449ac9c73756c29fa84191298c3aa73cb60dd7d/packages/preview-middleware-client/src/cpe/connector-service.ts#L47. We probably should have a list of changes that require reload in one place as well, but that could be done in a separate PR.

Reply:
we will consider this point in our upcoming PR

Reply:
@nikmace  thanks, I actually overlooked the 1st point. I have pushed the change.